scripts/awscli.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+set -e
+
+AWSCLI_VERSION="${AWSCLI_VERSION:-2.9.15}"
+
+missing=""
+command -v curl >/dev/null || missing="${missing} curl"
+command -v unzip >/dev/null || missing="${missing} unzip"
+test -d /etc/ssl/certs || missing="${missing} ca-certificates"
+
+if [ "$missing" != "" ]; then
+    apt-get update -qq
```

Comment:
Q: do we have any non-debian/apt images to worry about?

Reply:
Alpine images don't support awscli@v2, but have an `aws-cli` package for the few that we have. We don't have enough Red Hat images at the moment to worry about adding support for it here

Reply:
Perfect! Thank you for the info.